master/reference/networking-calico/implementation-notes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Implementation notes
```

Comment:
Not sure this belong in docs.projectcalico.org - it should probably be a markdown file in the networking-calico repository.

These are meant to be user-facing docs, so detailed implementation stuff doesn't belong.